components/ui/Loading.js

```diff
@@ -0,0 +1,47 @@
+import styled, { css, keyframes } from 'styled-components'
+
+const size = 1.6
+const sizeRem = `${size}rem`
+const beforeAfterOffset = size * 2
+const defaultBackgroundColor = '#8f8d76'
+
+const flashKeyframes = (props, percentages) => keyframes`
+  ${percentages} {
+    background-color: ${props.theme.colors.vibrantPixel};
```

Comment:
The original suggestion was to change this to:
```
background-color: ${props => props.theme.colors.vibrantPixel};
```

But we don't have direct access to `props` in `keyframes` - I believe because they are considered global declarations. So i needed to pass props from the call to `css` below and pass it through.